### ISSUE TITLE
Doom ajustment suggestion

### ISSUE THREAD

**Issue by HugMyPug** [TotalFreedom/TotalFreedomMod#1931]
Per seeing another issue, I saw how Ryan said that "Doom is supposed to be the last view of the user on the server" So, what I am suggesting is basically a user gets permbanned automatically by IP, the reason I am saying just IP is so if a person with a famous name gets permbanned, the name won't get permbanned.
Or maybe even add a blacklist to famous names being added to name permban..

Thoughts?

**Comment by mayokek** [comment]
It's already a thing...
https://github.com/TotalFreedom/TotalFreedomMod/blob/server/src/main/resources/config.yml#L258-L297

**Comment by ghost** [comment]
@HugMyPug this issue is invalid, when a user is doomed their name and ip are banned permanently from the server, however if the name is on that list it will remain unbanned. Invalid.

**Comment by Wild1145** [comment]
@Commodore64x That is incorrect. You are simply removed from admin, and then banned. The permban list is only modified when someone with FTP access modifies the file

**Comment by ghost** [comment]
What do you call  ban.addIp(playerIp); then??? I think that classifies as IP banning....

**Comment by Wild1145** [comment]
@Commodore64x A temp ban by IP... Not a permban as per the original issue...

**Comment by ghost** [comment]
@Wild1145, as I remember, it permbans you until you're /glist unbanned or someone wipes the banlist.

**Comment by Wild1145** [comment]
@Anturity That is not a permban, that does the same as /gtfo or /glist ban would.
